src/renderers/shared/stack/reconciler/ReactChildReconciler.js

```diff
@@ -22,11 +22,7 @@ var warning = require('fbjs/lib/warning');
 
 var ReactComponentTreeHook;
 
-if (
-  typeof process !== 'undefined' &&
-  process.env &&
-  process.env.NODE_ENV === 'test'
-) {
```

Comment:
Should i revert this change and the latter? Would be cool to fix this whole thing in `15.x`